quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/update/AbstractQuarkusCliUpdateIT.java

```diff
@@ -0,0 +1,84 @@
+package io.quarkus.ts.quarkus.cli.update;
+
+import static io.quarkus.test.util.QuarkusCLIUtils.getQuarkusAppVersion;
+import static io.quarkus.test.utils.AwaitilityUtils.untilAsserted;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import jakarta.inject.Inject;
+
+import org.apache.http.HttpStatus;
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.QuarkusCliClient;
+import io.quarkus.test.bootstrap.QuarkusCliRestService;
+import io.quarkus.test.logging.Log;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+import io.quarkus.test.services.quarkus.model.QuarkusProperties;
+import io.quarkus.test.util.DefaultQuarkusCLIAppManager;
+import io.quarkus.test.util.IQuarkusCLIAppManager;
+import io.quarkus.test.utils.AwaitilityUtils;
+
+@QuarkusScenario
+@DisabledOnNative // Only for JVM verification
+@Disabled("https://github.com/quarkusio/quarkus/issues/42567")
```

Comment:
Right now all tests added in this PR are disabled. However https://github.com/quarkusio/quarkus/issues/42567 is already fixed. IIRC we planned (or already have?) some Jenkins job that runs periodically against 3.8 SNAPSHOT. Maybe you could just disable this for 3.8.5 and lower so that we don't have to remember it is disabled when 3.8.6 is released.

Reply:
That makes sense. I changed it to work on 3.8 versions that are greater that 3.8.5.
I looks a bit strange to have Disabled and Enabled annotations just next to each other,  but I didn't figured out easy to read Enable regex that would match this case, even for possible future versions like 3.8.12 or something like that.

Reply:
> I looks a bit strange to have Disabled and Enabled annotations just next to each other, but I didn't figured out easy to read Enable regex that would match this case, even for possible future versions like 3.8.12 or something like that.

I don't mind, let's keep it.